src/scanoss/scanossgrpc.py

```
@@ -22,50 +22,72 @@
+    """
+
+    pass
+
+
 class ScanossGrpc(ScanossBase):
     """
     Client for gRPC functionality
     """
 
-    def __init__(
+    def __init__(  # noqa: PLR0913
```

Ruff (0.8.2)

Too many statements (51 > 50)

(PLR0915)